bundles/AdminBundle/Resources/public/js/pimcore/object/classes/data/reverseObjectRelation.js

```diff
@@ -122,7 +122,7 @@ pimcore.object.classes.data.reverseManyToManyObjectRelation = Class.create(pimco
                     rootProperty: "availableFields"
                 },
                 extraParams: {
-                    types: 'manyToManyObjectRelation',
+                    types: 'manyToManyObjectRelation,manyToOneRelation',
```

Comment:
Is it also possible with manyToManyRelation?

Reply:
Yes. It is also possible with manyToManyRelation but as we use manyToManyObjectRelation for objects that's why I didn't include manyToManyRelation.